docs/_docs/ignite-cli-tool.adoc

```diff
@@ -13,27 +13,72 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-== Ignite CLI Tool
+= Ignite CLI Tool
 
 The Ignite CLI tool uses REST API as a communication protocol between the node and the CLI,
-allowing you to configure the node.
+allowing you to configure the node. By default, the CLI tool runs in the REPL (Read Eval Print Loop) mode, but you can also execute commands without entering it.
 
 Below is a list of Ignite CLI tool commands with short descriptions:
 
-[cols="1,3",opts="header", stripes=none]
+== Command Options
+
+The following commands can be executed in REPL mode of the command line tool:
+
+[cols="1,1,3",opts="header", stripes=none]
 |===
-| Command | Description
-| `config` | Inspects and updates Ignite cluster configuration.
-| `config get` | Gets current Ignite cluster configuration values.
-| `config set` | Updates Ignite cluster configuration values.
-| `init` | Installs Ignite core modules locally.
-| `module` | Manages optional Ignite modules and external artifacts.
-| `module list` | Shows the list of available optional Ignite modules.
-| `module add` | Adds an optional Ignite module or an external artifact.
-| `module remove` | Adds an optional Ignite module or an external artifact.
-| `node`| Starts, stops, and manages locally running Ignite nodes.
-| `node start` | Starts an Ignite node locally.
-| `node stop` | Stops a locally running Ignite node.
-| `node classpath` | Shows the current classpath used by the Ignite nodes.
-| `node list` | Shows the list of currently running local Ignite nodes.
+| Command| Parameters | Description
+| `bootsrap` || Installs Ignite core modules locally. Previously `init`.
+| `clear` || Clears the terminal.
+| `cli config get` | `<config key-value>` | Gets the value of the specified CLI tool configuration.
+| `cli config set` | `<config key-value>` | Sets the value of the specified CLI tool configuration.
+| `cluster init` | `[--cluster-url] <--meta-storage-node> [--cmg-node]`| Initializes the cluster on the specified URL.
+| `cluster show` | `[--cluster-url]` | Displays the status of the specified cluster.
+| `cluster config show` | `[--cluster-url] [--selector]` | Gets the configuration of the specified cluster.
+| `cluster config update` | `[--cluster-url] <config in HOCON format or path to file>` | Updates the configuration of the cluster with the provided configuration in HOCON format.
+| `connect` | [--cluster-url] | Connects to the specified cluster.
```

Comment:
You can add that given empty `--cluster-url` will connect to localhost with default port.

Reply:
Also, when a user is connected all commands will use the connected node url as a default `--cluster-url` or `--node-url`